airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -156,7 +156,7 @@ def path(self, **kwargs) -> str:
 
     def request_params(self, **kwargs) -> MutableMapping[str, Any]:
         params = super().request_params(**kwargs)
-        params["types"] = "public_channel"
+        params["types"] = "public_channel,private_channel"
```

Comment:
The change is valid but without better improvement in documentation or giving options to users it can expose data users don't want to retrieve.
My suggestion is to add a boolean option if the user want to also retrieve private channels too.